### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by LeoColman**
Hi guys!

We've just started using Korge, and this bit in the documentation was a little confusing.

We made the sample compile and added the imports.

### CODE REVIEW

File: korge/reference/input/index.md [hunk 1]
```diff
@@ -39,12 +39,17 @@ view.addUpdater {
...
-    val scale = dt / 16.hrMilliseconds
-    if (input.keys[Key.LEFT]) x -= 2.0 * scale
+import com.soywiz.klock.milliseconds
+import com.soywiz.klogger.Console
+import com.soywiz.korev.Key
+
+
+view.addUpdater { timespan: TimeSpan ->
+    val scale = timespan / 16.milliseconds
+    if (input.keys[Key.LEFT]) x -= 2.0 * scale  // Same as `input.keys.pressing(Key.LEFT)`
```
Comment (LeoColman) [inline korge/reference/input/index.md:51]: Although the comment says that both are the same, I'd like to understand better why `get` == `pressing`. I had to look at the code as I couldn't assume this from the `get` alone.

**Comment by FSaurenbach** [comment]
Why should there be inports its a reference article. But i'm asking soywiz.

**Review by FSaurenbach — APPROVED** [review]

**Review by soywiz — APPROVED** [review]
I'm fine with that, since we are not hiddin them, looks reasonable to me to strip them since the IDE can find the right imports 👍

**Comment by FSaurenbach** [comment]
> I'm fine with that, since we are not hiddin them, looks reasonable to me to strip them since the IDE can find the right imports +1
Buts not in ecery other article

**Pull request merged**